replica/client/client.go

```diff
@@ -124,7 +124,7 @@ func (c *ReplicaClient) Revert(name, created string) error {
 	if err != nil {
 		return err
 	}
-
+	c.httpClient.Timeout = 0
```

Comment:
Setting timeout to 0 means this request may hung forever, till the connection is available.

Reply:
Why this change in revert?

Reply:
If the connection is available, we expect the server to respond. This is a revert operation and all the replicas should respond with success or failure before further steps could be taken.

Reply:
Revert also does a preload operation which may time out. Therefore this change was necessary.